wdl2cwl/main.py

```diff
@@ -426,6 +426,9 @@ def convert(workflow: str) -> str:
     if ast.task_parameter_meta_check:
         print("----WARNING: SKIPPING PARAMETER_META----")
 
+    if ast.task_meta_check:
+        print("----WARNING: SKIPPING META----")
+
```

Comment:
Is there a test that confirms this warning is printed?

Reply:
I'm not sure why UmiCorrection.wdl won't work for this. It has a meta section.

Reply:
Use https://docs.pytest.org/en/6.2.x/reference.html#std-fixture-capsys to check that the warning is printed